pkg/query/api/v1.go

```diff
@@ -350,17 +346,18 @@ func (api *API) labelValues(r *http.Request) (interface{}, []error, *apiError) {
 	if !model.LabelNameRE.MatchString(name) {
 		return nil, nil, &apiError{errorBadData, fmt.Errorf("invalid label name: %q", name)}
 	}
+
 	var (
-		warnmtx  sync.Mutex
-		warnings []error
+		warnmtx             sync.Mutex
+		warnings            []error
```

Comment:
That doesn't seem right – did you run `go fmt`?

Reply:
Seem that I forgot.. eh need to setup some commit pre hook finally

Reply:
Or a save hook in your editor :)

Reply:
I can't easily.. I have to many different repos with different formatting ):